### DISCUSSION TITLE
Where can I find the log of the log?

### DISCUSSION THREAD

**Discussion by andoniripollpro** [open-telemetry/opentelemetry-dotnet#4914] [Q&A]
Hi.

I am gonna write from another aproach: I launch the console on otlp option. It says that it's sending telemetry. But telemetry never arrives. And I don't find the log where explains that something is wrong. It looks like it is a SSL/TSL problem, but that's a conjecture base on a curl that I just did.

Here normal call to consoloe app with no error sings. Then a curl showing the likeliy cause of the problem (it is in spanish but the SSL/TSL error can be undestood), but never sure.
![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/9254576/c593ae10-b900-4573-8a91-5474b4f0e041)

Any help, please?

Regards,
Andoni

**Comment by cijothomas** [comment]
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/README.md#self-diagnostics

**Reply by andoniripollpro** [reply]
Thanks. I didn´t see it on the help
